### PR TITLE
Update package naming

### LINKED ISSUE #13: Consider renaming package
Project name is `accelerator-initializer` and the package said `com.hopper`. IMHO, package name should be `com.scotiabank.accelerator.initializer` or `com.scotiabank.initializer`. WDYT?

### PR DESCRIPTION
**Pull request by yshewchuk**
Now follows a more standard package naming structure

Fixes #13

### CODE REVIEW

**Comment by dkuwahara** [comment]
Actually, I saw after I approved some packages are

`com.scotiabank.accelerator.initializer.initializer` should fix those before merging

File: src/main/java/com/scotiabank/accelerator/initializer/controller/ComponentController.java [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
  */
-package com.hopper.controller;
+package com.scotiabank.accelerator.initializer.controller;
 
-import com.hopper.controller.request.ComponentAddRequest;
-import com.hopper.initializer.FileProcessor;
-import com.hopper.initializer.ProjectCreationService;
-import com.hopper.initializer.model.ProjectCreation;
+import com.scotiabank.accelerator.initializer.controller.request.ComponentAddRequest;
+import com.scotiabank.accelerator.initializer.initializer.FileProcessor;
```
Comment (dkuwahara) [inline src/main/java/com/scotiabank/accelerator/initializer/controller/ComponentController.java]: package name

Comment (eddumelendez) [reply src/main/java/com/scotiabank/accelerator/initializer/controller/ComponentController.java]: My suggestion for `com.scotiabank.accelerator.initializer.initializer` is `com.scotiabank.accelerator.initializer.core`. WDYT?

Comment (yshewchuk) [reply src/main/java/com/scotiabank/accelerator/initializer/controller/ComponentController.java]: That sounds good to me. Making the change

**Pull request merged**